api/services/roundService.js

```diff
@@ -0,0 +1,32 @@
+
+export default class RoundService {
+    constructor(players) {
+        this.players = players;
+
+    }
+
+    newRound() {
+        getCzar(newGame)//TODO: newgame is a boolean that is passed in
+
+    }
+
+    endRound() {
+
+    }
+
+    getCzar(newGame) {
+        if (newGame) {
+            this.currentCzarNum = Math.floor(Math.random() * Math.floor(this.players.length));
```

Comment:
Do you need to call `Math.floor()` on `this.players.length`? When will `this.players.length` ever not be an integer?